src/braket/pennylane_plugin/braket_device.py

```diff
@@ -342,6 +352,42 @@ def _check_supported_result_types(self):
             result_type.name for result_type in supported_result_types
         )
 
+    def _validate_noise_model_support_local_device(self):
+        supported_operations = [
+            ops.lower().replace("_", "")
+            for ops in self._device.properties.action[DeviceActionType.OPENQASM].supportedOperations
+        ]
+        noise_instructions = [
+            noise_instr.noise.name.lower().replace("_", "")
+            for noise_instr in self._noise_model._instructions
+        ]
+        if not all([noise in supported_operations for noise in noise_instructions]):
+            raise ValueError(
+                f"{self._device.name} does not support noise or the noise model includes noise "
+                + f"that is not supported by {self._device.name}."
+            )
```

Comment:
No need for a separate case for the local simulator; the pragma validation should also work here.